ArraysHomework/ArraysHomework.cs

```diff
@@ -70,14 +79,15 @@ public void Arrays()
             {
                 array1[up] = up;
                 array2[down] = down;
-                
             }
 
             // Your code here: combine array1 and array2 into an array named combined
             // Everything from array1 should be in the first half of combined, and everything
             // from array2 should be in the second half of combined.
-            
-            
+            int[] combined = new int[50];
+
+            combined = array1.Concat(array2).ToArray();
```

Comment:
This is a good answer, but it's getting a bit ahead of the topic.

How would you do this without using Concat() or ToArray()? Just using the flow control, array access/assignment, etc. statements we covered?